File: z2p7/C++/Account.cpp
#include<iostream>
using namespace std;
class Account{
    public:
        string accountHolder;
        string accountNumber;
        int    accountBalance;
        void createAccount(string accountHolder,string accountNumber){
            accountHolder = accountHolder;
            accountNumber = accountNumber;
            accountBalance= 1000;
        }
        void deposit (int depositAmmount){
            accountBalance = accountBalance + depositAmmount;
        }
        bool withdraw(int withdrawAmmount){
            if (withdrawAmmount >accountBalance)
            {
                return false;
            }else{
                return true;
            }
            
        }
        void showData(){
            cout<<"________Current Info__________"<<endl;
            cout<<"Account Holder Name :"<<accountHolder<<endl;
            cout<<"Account Number :"<<accountNumber<<endl;
            cout<<"Account Balance :"<<accountBalance<<endl;
        }
};
int main(){
    Account account1;
    account1.showData();
    return 0;
}

File: z2p7/z2p7phthon/Account.py
class Account:
    def __init__(self,name,account_number,initial_amount):
        self.name = name
        self._no = account_number
        self._balance = initial_amount
    def deposit(self,amount):
        self.balance += amount
    def withdraw(self,amount):
        self.balance -= amount
    def dump(self):
        print('%s,%s balance : %s' %(self.name,self.no,self.balance))
a1 =Account('Ma Ma','0123456',20000)
print(a1.__dict__)
a1.dump()
a1.deposit(50000)
a1.withdraw(69000)
a1.dump()
        